Features scss footer and guarantee version2

Kate-Kompani opened the pull request.

Kate-Kompani committed:
mobile_version_of_footer(draft)

Kate-Kompani committed:
scss/finished/footer_and_guarantee

Vlad-Nemanikhin reviewed (commented):
Пропиши нормальные пути свг. У тебя они работают в деплое?
images/путь

В футере не нужно задавать height. Падингами и контентом набрать нужный размер по высоте.
В остальном норм

Kate-Kompani committed:
features/scss/footer_guarantee

1) убрала высоту в футере, задала падингами
2) проверила путь к картинкам - исправила. Локально - ошибок не выдает
